### ISSUE TITLE
Calico installation doesn't work

### ISSUE THREAD

**Issue by kellianoy** [projectcalico/calico#6092]
# Calico installation doesn't work

Hello, I have a Kubernetes cluster that I create with kubeadm.

Installing the elements of the quickstart:

- kubectl create -f https://projectcalico.docs.tigera.io/master/manifests/tigera-operator.yaml
- kubectl create -f https://projectcalico.docs.tigera.io/master/manifests/custom-resources.yaml

tigera-operator works as intended by creating the pods, however for `custom-resources`, I get the following messages:

```bash
'Warning: policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+\n' +
    'unable to recognize "calico/custom-resources.yaml": no matches for kind "Installation" in version "operator.tigera.io/v1"\n' +
    'unable to recognize "calico/custom-resources.yaml": no matches for kind "APIServer" in version "operator.tigera.io/v1"\n',
```

## Expected Behavior

The pods should be created accordingly to the manifest.

## Current Behavior

`custom-resources` doesn't work.

## Steps to Reproduce (for bugs)

Basically, follow the quickstart:

1. kubeadm init --pod-network-cidr=10.10.40.1/24
2. Change custom-resources ip address to fit the one above
3. Apply the manifests

## Context

I want to install Calico and Calicoctl to use them with MetalLB

## Your Environment

* Calico version: Nightly
* Orchestrator version (e.g. kubernetes, mesos, rkt): kubernetes 1.23.6
* Operating System and version: Centos 7

**Comment by kekekekekeshi** [comment]
请问custom-resources在1.25.2中应该怎么写呢？
